docker/Dockerfile.base

```diff
@@ -0,0 +1,25 @@
+FROM debian:jessie-slim
+
+WORKDIR /papers
+
+RUN apt-get update -y && apt-get install -y \
+    wget \
+    libpod-pom-perl \
+    fontconfig \
+    fonts-lmodern \
+    && rm -rf /var/lib/apt
+
+RUN echo 'aacaa35130327cd8e2c8914732d30f0b  install-tl-unx.tar.gz' > texlive.md5
```

Comment:
Add comment like
`# Install a small version of TexLive with the profile in texlive.profile`